client/src/components/shared/LogoutModal.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * LogoutModal component display a modal and ask for confirmation when the sign out button is clicked 
+ */
+import { Button, Modal } from 'react-bootstrap';
+import { useSelector, useDispatch } from 'react-redux';
+import { LOG_OUT } from '../../actions/types';
+import { REMOVE_LOG_OUT_ALERT } from '../../actions/types';
+
+const LogoutModal = () => {
+    const logoutAlert = useSelector((state) => state.auth.logoutAlert);
```

Comment:
curious: do we really need `REMOVE_LOG_OUT_ALERT`? Why not use whether the user has successfully logged out as indication of whether we should remove the alert?
E.g. maybe use this `LOG_OUT`?

Reply:
Same for `DISPLAY_LOG_OUT_ALERT` - I wonder if we can implement this functionality without adding to our redux